.github/label-actions.yml

```
@@ -124,6 +124,22 @@
     Read the [Renovate docs, Troubleshooting](https://docs.renovatebot.com/troubleshooting/) to learn more about getting the docs, and getting the correct type of logs.
 
 
+    Thanks, the Renovate team
+
+'auto:logs-reduction':
+  comment: >
+    Hi there,
+
+
+    Please try to reduce the amount of logs you're pasting into this discussion. Maintainers have a limited amount of time to help you, and often do so from mobile devices, so it's easier for us to help you if you provide only the relevant parts of the logs and/or make a good attempt to look through them first and point to the lines you think are relevant.
+
+
+    As an example, if your problem is about a certain dependency, find the log sections which apply to that dependency and paste only those sections. Similarly, if your problem is about a particular branch/PR, find the log sections which apply to that branch/PR and paste only those sections.
+
+
+    If you're unsure, it's also perfectly acceptable to paste the full logs, including into a gist, but if that's the only thing you do, it's likely to take longer for us to help you or we may not start.
```

```suggestion
    If you're not sure, it's acceptable to paste the full logs, including into a gist. Please try to explain the problem in enough detail to give us starting points to debug. If you only paste the full log, and do nothing else, it is likely that we will take longer to help you, or we may not start to help you at all.
```

I think the goal here is to:

- Paste full log if you have no idea what's most relevant
- But please give us more info to help you
- If you only "log dump", then we may take longer to help you, or not start to help you at all

I'm open to suggestions on the tone of the message! 😉